apps/galaxy/src/modules/auth/SignUpForm/StaffInfoPage.tsx

```diff
@@ -7,12 +7,10 @@ import useSignUp from "@src/services/signup/useSignUp";
 import { Dispatch, SetStateAction } from "react";
 
 const StaffInfoPage = (props: {
```

Comment:
You can deconstruct data here if you are not reusing `props` in other area

Reply:
Do you mean this:
![image](https://user-images.githubusercontent.com/73293255/200097255-6f80c3bd-5dc9-4985-8eca-7a23f89762d4.png)